adis16350:

#komunikacne parametre
 port_name: /dev/ttyUSB0
 baudrate: 115200
 stop_bits: 1    #1 - stopBits 1  #2 - stopBits 2 #3 - stopBits 1.5 
 parity: 0       #0 - none #1 - odd parity #2 - even parity
 byte_size: 8    #5 - fivebits #6 - sixbits #7 - sevenbits #8 eightbits
 
 #nastavenia zakladnych registrov
 
 #register SENS_AVG 0x38 0x39
 digital_filtering: 6 #rozsah 0-6
                      #sin ( π × N × f × t s )/N × sin ( π × f × t s ), N - filter taps
                      #N = 2^M, priklad pre hodnotu 3 N = 2^3 = 8 filter taps
 range: 3             #1 - 75°/s, filter taps ≥ 16
                      #2 - 150°/s, filter taps ≥ 4
                      #3 - 300°/s (default condition)

 #register SMPL_PRD 0x36 0x37                                        
 sample_rate: 1    #Ts = Tb * (Ns + 1); rozsah 0-15
                   #pre 0-7 Tb = 0.61035 ms a Ns = hodnota z parametra
                   #pre  8-15 Tb = 18.921 ms a Ns = hodnota z parametra - 7
                   #priklad pre hodnotu 1 bude Ts = 0.61053 * (1 + 1)
                   
 #register MSC_CTRL 0x34 0x35
 msc_ctrl: 3 #0 - nic
             #1 - Linear accelerometer origin alignment
             #2 - Linear acceleration bias compensation for gyroscopes
             #3 - vsetko
             
 #kalibracia
 calibration_type: 5 #zaporne cislo, zmaze nastaveni offset na Z gyro a nebude nic kalibrovat
                     #0 - ziadna kalibracia
                     #0. bit = 1 - pouzije sa auto kalibracia (kratka)
                     #1. bit = 1 - pouzije sa precisna kalibracia (dlha - treba nechat senzor cca 30 sekund vklude)
                     #2. bit = 1 - 2000 krat sa precita vzorka z Gyra Z a vypocita sa offset, ktory sa zapise do registra ZGYRO_OFF (asi najpresnejsie)
                     #nastavenim viacerych bitov na 1 je mozne kombinovat. Napr. pre hodnotu 5 (binarne 101) sa spusti kratka kalibracia a nasledne kalibracia Z gyra.
                     
 #parametre pre softver 
 use_complementary_filter: True
 using_gyro: 0.98
 using_akcel: 0.02
 use_moving_average: True
 data_topic_name: imu_data
 status_period: 0.5
 publish_rate: 150
